Add README on how to monitoring etcd cluster (#5)
SUSE/caasp-monitoring: README.md

# CaaSP Grafana Monitoring Dashboards
Example files for SUSE CaaS Platform Grafana monitoring dashboards.

## Monitor ETCD Cluster

> Install [etcd-cluster.yaml](grafana-dashboards-caasp-etcd-cluster.yaml) to monitoring the etcd cluster. But we need to manually setup extra configuration to Prometheus server.

Reviewer: ```suggestion
Install [etcd-cluster.yaml](grafana-dashboards-caasp-etcd-cluster.yaml) to monitor the etcd cluster and also need to manually setup extra configuration to Prometheus server.
```
